heureka/ui/src/components/services/ServicesList.jsx

```diff
@@ -27,8 +27,10 @@ const ServicesList = ({ items, isLoading }) => {
           <DataGridHeadCell>Service</DataGridHeadCell>
           <DataGridHeadCell>Owners</DataGridHeadCell>
           <DataGridHeadCell>Support Groups</DataGridHeadCell>
-          <DataGridHeadCell>Component Instances Count</DataGridHeadCell>
-          <DataGridHeadCell>Total Vulnerability Matches</DataGridHeadCell>
+          <DataGridHeadCell>
+            Total Number of Component Instances
```

Comment:
In general I would advise not to use these super long Headings. It makes the table render weirdly if the content in the cells is a lot smaller than the heading for the column. If the long text is essential then my suggestion is to use a short version of the text as the heading and add a tooltip to it with the long form version.